cmd/commands/subscription_display.go

```diff
@@ -0,0 +1,97 @@
+package commands
+
+import (
+	"fmt"
+	"github.com/knative/eventing/pkg/apis/channels/v1alpha1"
+	"io"
+)
+
+type subscriptionDisplay struct {
+	headers       *[]subscriptionHeader
+	subscriptions *[]v1alpha1.Subscription
```

Comment:
I think this should be an array of pointers, instead of a pointer to an array.
See eg the hoops you're going through on line 34